config/docs.js

```diff
@@ -19,7 +19,7 @@ module.exports = [
     githubRepo: 'apache/apisix-ingress-controller',
     version: '1.6.0',
     releaseDate: '2022-12-30',
```

Comment:
Should these 2 be updated as well? Loop @SkyeYoung  @juzhiyuan @AlinsRan 

https://github.com/apache/apisix-ingress-controller/releases/tag/2.0.0
```suggestion
    version: '2.0.0',
    releaseDate: '2025-12-17',
```